app/Policies/TransactionPolicy.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+namespace App\Policies;
+
+use App\Transaction;
+use App\User;
+use Illuminate\Auth\Access\HandlesAuthorization;
+
+class TransactionPolicy
+{
+    use HandlesAuthorization;
+
+    public function delete(User $user, Transaction $transaction)
+    {
+        return $user->hasRole(Role::STUDENT_COUNCIL);
+    }
```

Comment:
For later usage, deleting transactions would be authorized for the receiver user (until the transaction will not be in the checkout), or the checkout's manager (economic committe leader / admin leader)

Reply:
Ok for now.